shim/src/main.rs

```diff
@@ -446,6 +447,10 @@ fn get_version() -> &'static str {
 }
 
 fn main() -> Result<()> {
+    let mut updater =
+        UpdateNotifier::new(String::from("turbo"), Some(String::from("latest")), None);
+    updater.check();
```

Comment:
I'm intentionally not handling the response here. I haven't decided if this should return an error, or if it should just never (purposely) panic. I'm leaning towards the latter since we don't want this to interrupt normal execution. However we could return here and let the consumer decide.